plugins/bf4/bf4.cpp

```diff
@@ -70,25 +100,22 @@ static int fetch(float *avatar_pos, float *avatar_front, float *avatar_top, floa
     }
 
     serverid[sizeof(serverid)-1] = 0; // NUL terminate queried C strings. We do this to ensure the strings from the game are NUL terminated. They should be already, but we can't take any chances.
-    std::string ServerID(serverid);
-    ServerID = jsonStringEscape(ServerID); // Replace '"' character with ' ' (space) to prevent JSON break.
+    escape(serverid);
     std::ostringstream ocontext;
-    ocontext << " {\"Server ID\": \"" << ServerID << "\"}"; // Set context with server ID
+    ocontext << " {\"Server ID\": \"" << serverID << "\"}"; // Set context with server ID
     context = ocontext.str();
 
     std::wostringstream oidentity;
     oidentity << "{";
     host[sizeof(host)-1] = 0; // NUL terminate queried C strings. We do this to ensure the strings from the game are NUL terminated. They should be already, but we can't take any chances.
-    std::string Host(host);
-    Host = jsonStringEscape(Host);
-    if (Host.find("bot") == std::string::npos) // Set host string as empty if "bot" is found in it.
+    escape(host);
+    if (strstr(host, "bot") == NULL) // Set host string as empty if "bot" is found in it.
```

Comment:
This comment isn't correct. It wasn't before either.

What's the idea, @davidebeatrici?